.golangci.yml

```diff
@@ -1,6 +1,6 @@
 run:
   timeout: 10m
```

Comment:
Let's please also bump:
* go.mod files

In a follow-up:
* cloudbuild.yaml, cloudbuild-nightly.yaml: there is currently no gcb-docker-gcloud image with Go 1.20 => I think that was in test-infra and we can open a PR to bump Go in that image

Reply:
Noted.